fabtests/prov/efa/src/rdm_ep_rnr_retry.c

```diff
@@ -26,9 +26,45 @@
  * CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
  * SOFTWARE.
  *
- * This program tests the RNR retry counter reset via fi_setopt.
- * When running the test, use `-R` option to specify RNR retry counter.
- * The valid values are 0 - 7 (7 inidicates infinite retry on firmware).
+ * This program has two tests:
+ *
+ * 1. poll_rnr_cq_error() tests the RNR retry counter reset via fi_setopt.
+ *    When running the test, use `-R` option to specify RNR retry counter.
+ *    The valid values are 0 - 7 (7 inidicates infinite retry on firmware).
+ *
+ * 2. rnr_queue_resend_test() tests RNR queue/re-send logic for all different
```

Comment:
I will read in more detail, but on a high level, this need to be a separate test application.

Reply:
Split into two tests (1) check to read RNR cq error entry (2) check RNR queue/re-send logic  in new rev.